db/import_test.go

```diff
@@ -24,6 +24,55 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestFeedImport(t *testing.T) {
```

Comment:
Since migrate isn't a metadata-only update (it modifies the document body) and doesn't modify _mou, I didn't think additional test cases for specifically for migrate (inline sync metadata) were necessary.

I agree about the on-demand import case, I'll add that.